inst/testfiles/ax6_testfile_formatted_timestamps.csv

```diff
@@ -0,0 +1,11320 @@
+2019-12-23 21:04:06.690,0.007324,0.071289,0.008789,0.274658,-0.503540,15.769958

```

Comment:
- File ax6_testfile_formatted_timestamps.csv has 11300 rows and is 917 kb. Is this necessary or could the same functionality be tested with a file that has only 100 lines? Same question for ax3_testfile_unix_timestamps.csv.
- I am trying to be considerate with CRAN's storage space, especially because this quickly adds up when working with multiple files.
- On that note, I think  ax3test.wav can be omitted?

Reply:
Good point. Will shorten the csv files, and I'll remove ax3test.wav too.

Reply:
Shortened both csv files to 210 lines each. That's the smallest length for which g.readaccfile() won't return filetooshort at the sample rate of these recordings.